src/main/java/org/spongepowered/common/entity/player/LoginPermissions.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.entity.player;
+
+public final class LoginPermissions {
+
+    public static final int BYPASS_WHITELIST_LEVEL = 1;
+
+    public static final String BYPASS_WHITELIST_PERMISSION = "minecraft.login.bypass-whitelist";
```

Comment:
Is the permission native to Minecraft or Sponge?

Reply:
I'm pretty sure there is no such permission in Minecraft

Reply:
Then it should be sponge based.

Reply:
Bypassing login and whitelist is a minecraft feature, we just modify it to be a permission. The same is done for the selector and command block permissions. This is different for the help command permission since a custom 'sponge' based help command is provided instead.

Reply:
On second thought, this is consistent with minecraft.commandblock and minecraft.selector.